Source/DataModels/Windows/SubWindows/ComponentWindows/WindowComponentText2D.cpp

```diff
@@ -0,0 +1,53 @@
+#include "StdAfx.h"
+#include "WindowComponentText2D.h"
+
+#include "DataModels/Components/UI/ComponentText2D.h"
+#include "Resources/ResourceFont.h"
+#include "DataModels/Windows/EditorWindows/ImporterWindows/WindowFontInput.h"
+
+
+WindowComponentText2D::WindowComponentText2D(ComponentText2D* component) :
+	ComponentWindow("TEXT2D", component),
+	componentText2D(component),
+	inputFont(std::make_unique<WindowFontInput>()),
+	str1("")	
+{}
+
+WindowComponentText2D::~WindowComponentText2D()
+{
+}
+
+void WindowComponentText2D::DrawWindowContents()
+{
+	DrawEnableAndDeleteComponent();	
+		
+	std::string items;
+	std::string texts;
+	
+	for (std::shared_ptr<ResourceFont> i : componentText2D->getFonts())
+	{
+		items+=i.get()->GetFileName();
+		items += '\0';
```

Comment:
Shouldn't this be a line break?
```suggestion
		items += '\n';
```